reference/5.0/Microsoft.PowerShell.Archive/Compress-Archive.md

```diff
@@ -16,98 +17,130 @@ Creates an archive, or zipped file, from specified files and folders.
 ## SYNTAX
 
 ### Path (Default)
-```powershell
+
+```
 Compress-Archive [-Path] <String[]> [-DestinationPath] <String> [-CompressionLevel <String>] [-WhatIf]
  [-Confirm] [<CommonParameters>]
 ```
 
 ### PathWithUpdate
-```powershell
+
+```
 Compress-Archive [-Path] <String[]> [-DestinationPath] <String> [-CompressionLevel <String>] [-Update]
  [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ### PathWithForce
-```powershell
+
+```
 Compress-Archive [-Path] <String[]> [-DestinationPath] <String> [-CompressionLevel <String>] [-Force] [-WhatIf]
  [-Confirm] [<CommonParameters>]
 ```
 
 ### LiteralPathWithUpdate
-```powershell
+
+```
 Compress-Archive -LiteralPath <String[]> [-DestinationPath] <String> [-CompressionLevel <String>] [-Update]
  [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ### LiteralPathWithForce
-```powershell
+
+```
 Compress-Archive -LiteralPath <String[]> [-DestinationPath] <String> [-CompressionLevel <String>] [-Force]
  [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ### LiteralPath
-```powershell
+
+```
 Compress-Archive -LiteralPath <String[]> [-DestinationPath] <String> [-CompressionLevel <String>] [-WhatIf]
  [-Confirm] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The `Compress-Archive` cmdlet creates a zipped (or compressed) archive file from one or more specified files or folders.
-An archive file allows multiple files to be packaged, and optionally compressed, into a single zipped file for easier distribution and storage.
-An archive file can be compressed by using the compression algorithm specified by the CompressionLevel parameter.
 
-Because `Compress-Archive` relies upon the Microsoft .NET Framework API `System.IO.Compression.ZipArchive` to compress files, the maximum file size that you can compress by using `Compress-Archive` is currently 2 GB.
-This is a limitation of the underlying API.
+The `Compress-Archive` cmdlet creates a zipped (or compressed) archive file from one or more
+specified files or folders. An archive file allows multiple files to be packaged, and optionally
+compressed, into a single zipped file for easier distribution and storage. An archive file can be
+compressed by using the compression algorithm specified by the CompressionLevel parameter.
+
+Because `Compress-Archive` relies upon the Microsoft .NET API `System.IO.Compression.ZipArchive` to
+compress files. Therefor, the maximum file size that is 2 GB. This is a limitation of the underlying
+API.
 
 ## EXAMPLES
 
 ### Example 1: Create an archive file
+
+This command creates a new archive file, Draft.zip, by compressing two files, Draftdoc.docx and
```

Comment:
Files aren't in quotes here, but are in the next example. The representations should match.

Reply:
In example 2, the filenames require quotes. However, I code fenced them for emphasis and clarity.